backstop/tests/menu.html

```diff
@@ -629,7 +629,7 @@ <h3>Header</h3>
               ></svg-model>
               <span class="iui-button-label">
                 <div>Model Beta</div>
```

Comment:
Does this need to be changed to have a class name of `iui-header-breadcrumbs-button-name` like the Project Button example?

Reply:
Yeah, I think it does.  It should be updated in the latest push.

Reply:
I don't see the update? I thought these cases would be changed to `<div class="iui-header-breadcrumb-button-label-name">Model Beta</div>`